healthcheck/sqlhealthcheck.go

```diff
@@ -54,32 +61,89 @@ func ReadYamlFromFile(path string) Format {
 func RunHealthChecks(healthChecks Format, cxn *sql.DB) Format {
 
 	for _, healthCheck := range healthChecks.Tests {
```

Comment:
It might be a bit more readable to make this "_, test" instead of "_, healthCheck", because in other contects "healthChecks" refers to the whole yaml file (i.e. a Format instance).